tools/update-requirements.sh

```diff
@@ -20,7 +20,7 @@ python3 -m venv $TMPDIR
 source $TMPDIR/bin/activate
 
 pip3 install \
-  fontmake \
+  ufo2ft \
```

Comment:
oh I see. compreffor and cu2qu are required by ufo2ft, so they get pulled in by pip. It's fine then, leave them where they are, even if you don't use them